src/Plugin/ElasticsearchIndexBase.php

```diff
@@ -114,10 +115,17 @@ public function setup() {
       $index_name = $this->getIndexName([]);
 
       if (!$this->client->indices()->exists(['index' => $index_name])) {
-        $this->client->indices()->create([
+        $params = [
           'index' => $index_name,
           'body' => $index_definition->toArray(),
-        ]);
+        ];
+
+        // Create the index.
+        $operation = ElasticsearchOperations::INDEX_CREATE;
```

Comment:
Should we also try to catch an event exception here and do something about it? Or do we just let it bubble up to be catched later?

Reply:
I think failure to create an index should be visible and specifically handled.